### PR TITLE
Add repertoire filter

### PR DESCRIPTION
**Pull request by meyad-g**

### CODE REVIEW

File: app/assets/images/texture.png
Comment (sbsto) [inline app/assets/images/texture.png:1]: no

File: app/javascript/controllers/repertoire_selector_controller.js
Comment (sbsto) [inline app/javascript/controllers/repertoire_selector_controller.js:1]: would rather this uses the existing controller with the rest of the logic abstracted to ruby

File: app/views/repertoire/index.html.erb [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 	<%= turbo_frame_tag "body" do %>
-		<div class="mt-4 flex flex-col align-center items-center justify-center">
-			<div class="repertoire-container mb-12">
+		<div class="mt-2 -mx-8 pt-2 flex flex-col align-center items-center justify-center">
+			<div class="repertoire-container mb-4 flex justify-center items-center">
 				<div class="stream-counts">
 					<%= render "repertoire/repertoire_stream_counts", repertoire_songs: @repertoire_songs %>
 				</div>
 			</div>
+			<div class="sm:w-[1000px] flex justify-center md:justify-end md:pr-9 pb-6">
```
Comment (sbsto) [inline app/views/repertoire/index.html.erb:9]: don't like this!

File: app/assets/images/texture.png
Comment (meyad-g) [reply app/assets/images/texture.png:1]: gone

File: app/javascript/controllers/repertoire_selector_controller.js
Comment (meyad-g) [reply app/javascript/controllers/repertoire_selector_controller.js:1]: life isn't fair sometimes

File: app/views/repertoire/index.html.erb (hunk 1 above)
Comment (meyad-g) [reply app/views/repertoire/index.html.erb:9]: changing

**Pull request merged**